File: Alura/CarreiraCSharp/CarreiraCSharpAlura.Arquivos/Conta.cs
namespace CarreiraCSharpAlura.Arquivos
{
    public class Conta
    {
        public string Agencia { get; }

        public string Numero { get; }

        public decimal Saldo { get; }

        public string Titular { get; }

        public Conta(string agencia, string numero, decimal saldo, string titular)
        {
            Agencia = agencia;
            Numero = numero;
            Saldo = saldo;
            Titular = titular;
        }

        public override string ToString()
        {
            return $"Conta(Agencia={Agencia}, Numero={Numero}, Saldo={Saldo}, Titular={Titular})";
        }
    }
}

File: Alura/CarreiraJava/parte2/src/Conta.java

public class Conta {
	private double saldo;
	private String agencia;
	private String numero;
	private Cliente cliente;
	
	public Conta(String agencia, String numero, Cliente cliente) {
		super();
		this.agencia = agencia;
		this.numero = numero;
		this.cliente = cliente;
	}
	
	public double getSaldo() {
		return saldo;
	}
	
	public String getAgencia() {
		return agencia;
	}
	
	public String getNumero() {
		return numero;
	}
	
	public Cliente getCliente() {
		return cliente;
	}
}
